package.json

```diff
@@ -30,51 +31,23 @@
     "request-promise": "0.4.2"
   },
   "devDependencies": {
-    "babel-core": "6.14.0",
-    "babel-eslint": "6.1.2",
-    "babel-plugin-add-module-exports": "0.2.1",
-    "babel-plugin-transform-class-properties": "6.11.5",
-    "babel-plugin-transform-decorators-legacy": "^1.3.4",
-    "babel-plugin-transform-flow-strip-types": "6.14.0",
-    "babel-plugin-transform-runtime": "6.15.0",
-    "babel-preset-es2015": "6.14.0",
-    "babel-preset-es2015-rollup": "1.2.0",
-    "babel-preset-stage-0": "6.5.0",
-    "browserify": "11.1.0",
-    "eslint": "2.13.1",
-    "eslint-plugin-import": "1.3.0",
-    "gulp": "3.9.1",
-    "gulp-babel": "6.1.2",
-    "gulp-bump": "0.1.11",
-    "gulp-eslint": "2.0.0",
-    "gulp-istanbul": "0.5.0",
-    "gulp-jscs": "1.1.2",
-    "gulp-jshint": "1.8.4",
-    "gulp-load-plugins": "0.6.0",
-    "gulp-mocha": "1.1.0",
-    "gulp-plumber": "0.6.5",
-    "gulp-sourcemaps": "1.5.2",
-    "gulp-util": "3.0.1",
-    "should": "4.0.4",
-    "vinyl-source-stream": "1.1.0"
+    "@types/node": "6.0.60",
+    "@types/node-uuid": "^0.0.28",
+    "@types/request": "^0.0.39",
+    "@types/request-promise": "^4.1.33",
+    "ava": "0.17.0",
+    "tslint": "4.3.1",
+    "typescript": "2.1.5"
   },
   "scripts": {
-    "test": "gulp test",
-    "prepublish": "gulp compile"
```

Comment:
@bsouthga: Awesome. Does the `gulp pkg` task work for you in that branch (particularly, does it install deps into `/dist/node_modules`?). If so, ship it from my side. (You can use a stand in `--target` like test-prod since the task just builds the zip)